src/js/common/datetime.js

```diff
@@ -621,6 +621,13 @@ datetime = {
         date.setHours(24);
 
         return date;
+    },
+
+    getDiffDays: function(d1, d2) {
+        var _d1 = new TZDate(d1.getFullYear(), d1.getMonth(), d1.getDate()).getTime();
```

Comment:
Why don't you just use `d1.getTime() - d2.getTime()`. I think it's enough.

I made a similar function in v2.
```js
function getDateDifference(d1: TZDate, d2: TZDate) {
  return Math.floor((d1.getTime() - d2.getTime()) / MILLISECONDS_PER_DAY);
}
```

Reply:
```js
const d1 = new Date('2012-03-02T00:00:00')
// Fri Mar 02 2012 00:00:00 GMT+0900 (대한민국 표준시) 

const d2 = new Date('2012-03-01T03:00:00')
// Thu Mar 01 2012 03:00:00 GMT+0900 (대한민국 표준시)

```
expect `+1` but `0`

Reply:
The difference hours on your example is 21 hours not over 24 hours.  expected value is 0.

Reply:
The time is less than 24 hours, but the dates differ by one day.
The label displayed is not a time difference but a date difference.